code/game/objects/items/tools/crowbar.dm

```diff
@@ -157,27 +185,68 @@
 	playsound(src, 'sound/items/tools/change_jaws.ogg', 50, TRUE)
-	if(tool_behaviour == TOOL_CROWBAR)
+	if(tool_behaviour != TOOL_WIRECUTTER)
 		RemoveElement(/datum/element/cuffsnapping, snap_time_weak_handcuffs, snap_time_strong_handcuffs)
 	else
 		AddElement(/datum/element/cuffsnapping, snap_time_weak_handcuffs, snap_time_strong_handcuffs)
 	return COMPONENT_NO_DEFAULT_MESSAGE
 
-/obj/item/crowbar/power/syndicate
-	name = "jaws of death"
-	desc = "An improved, faster, and smaller copy of Nanotrasen's standard jaws of life. Can be used to force open airlocks in its crowbar configuration."
-	icon_state = "jaws_syndie"
-	w_class = WEIGHT_CLASS_SMALL
-	toolspeed = 0.5
-	force_opens = TRUE
+/*
+ * Signal proc for [COMSIG_JAWS_OF_LIFE_FORCE_OPEN_AIRLOCK].
+ *
+ * Determines if our jaws of life is restricted from opening some doors, and whether or not we need to alert over the radio whenever they are used to
+ * pry open a door. Useful if you want to restrict jaws of life in some fashion.
+ */
 
-/obj/item/crowbar/power/examine()
-	. = ..()
-	. += " It's fitted with a [tool_behaviour == TOOL_CROWBAR ? "prying" : "cutting"] head."
+/obj/item/crowbar/power/proc/on_force_open(obj/item/source, mob/user, obj/machinery/door/airlock/target)
+	SIGNAL_HANDLER
+
+	var/list/collective_access = list()
+	collective_access += target.req_access
+	collective_access += target.req_one_access
+
+	if(limit_jaws_access)
+		for(var/possible_blacklisted_access in collective_access)
+			if(possible_blacklisted_access in blacklisted_access)
+				playsound(src.loc, 'sound/machines/buzz/buzz-sigh.ogg', 50, FALSE)
+				user.balloon_alert(user, "cannot pry open!")
+				return COMPONENT_JAWS_DO_NOT_ALLOW
+
+	if(radio_alert && COOLDOWN_FINISHED(src, alert_cooldown))
+
+		if(!collective_access) //Return if the door has literally no access at all
+			return COMPONENT_JAWS_ALLOW
+
+		for(var/possible_public_access in collective_access) //Return if the door has otherwise unimportant access
+			if((possible_public_access in ignored_access))
+				return COMPONENT_JAWS_ALLOW
+
+		sound_the_alarms(user, target)
+	return COMPONENT_JAWS_ALLOW
+
+///Our alert for our jaws of life.
+/obj/item/crowbar/power/proc/sound_the_alarms(mob/user, obj/machinery/door/airlock/target)
+		aas_config_announce(/datum/aas_config_entry/jaws_entry_alert, list(
+			"PERSON" = user.name,
+			"LOCATION" = get_area_name(target),
+			"TOOL" = name), src, list(RADIO_CHANNEL_SECURITY), RADIO_CHANNEL_SECURITY)
+		COOLDOWN_START(src, alert_cooldown, alert_cooldown_time)
+
+/datum/aas_config_entry/jaws_entry_alert
+	// This tool screams into the radio whenever the user successfully pries open an airlock.
+	name = "Door Forced Entry Alert"
+	announcement_lines_map = list(
+		RADIO_CHANNEL_SECURITY = "SECURITY ALERT: %PERSON has forced open a door at %LOCATION using %TOOL. Confirm that this was done during an emergency by authorized staff.",
```

Comment:
Can we get an announcement in med channel so the CMO can know if their parameds are off tiding as well? 🙂

Reply:
👍